bench/gemm-benchmark.cc

```diff
@@ -143,7 +143,7 @@ void GEMMBenchmark(benchmark::State& state,
   std::generate(b.begin(), b.end(), std::ref(i32rng));
 
   const size_t w_element_size = sizeof(int8_t);
-  const size_t w_size = nc_stride * sizeof(int32_t) + kc_stride * nc_stride * w_element_size;
+  const size_t w_size = nc_stride * (sizeof(float) + sizeof(int32_t)) + kc_stride * nc_stride * w_element_size;
```

Comment:
Are you sure about the heap corruption issue, that this is the correct fix?

We run these tests with sanitizers, I'm pretty sure they would catch an out of bounds access that is fixed by a change like this. I'm concerned that this is hiding a bug in the new code?

Reply:
I looked into this, and I think you're right